test/local/account_routes.js

```diff
@@ -701,6 +701,20 @@ test('/account/login', function (t) {
       })
     })
 
+    t.test('location data is present in sign-in confirmation email', function (t) {
+      config.signinConfirmation.sample_rate = 1
+      return runTest(route, mockRequest, function (response) {
+        t.equal(mockMailer.sendVerifyLoginEmail.callCount, 1, 'mailer.sendVerifyLoginEmail was called')
+        t.equal(mockMailer.sendVerifyLoginEmail.getCall(0).args[2].location.city, 'Mountain View')
```

Comment:
We can put these location asserts for the `New Device Login Notification` as well, https://github.com/mozilla/fxa-auth-server/pull/1334/files#diff-fcdd8be5dbf433fe907739c4a898fceeR677